src/patternfly/components/Tabs/examples/tabs-secondary.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="pf-c-tabs-container">
+  {{#> tabs tabs--attribute='aria-label="Secondary Tabs Example"'}}
+    {{#> tabs-item tabs-item--attribute='id="ex3-tab1"'}}Tab item 1{{/tabs-item}}
+    {{#> tabs-item tabs-item--attribute='id="ex3-tab2"' tabs-item--current="true"}}Tab item 2{{/tabs-item}}
```

Comment:
Do you mean making "Secondary tab item 1" have `.pf-m-current`? That makes sense (done).

Do you think we should also make every "Tab item 1" have `.pf-m-current` rather than "Tab item 2" across all three examples? I suppose that would also be the typical default.

Reply:
👍

Reply:
👍